From acd4dc0023cbafe1357dcb651b07ceb4d0bd2e15 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:24 -0400
Subject: [infiniband] usnic: Remove unused variable in usnic_debugfs_exit()

Message-id: <84497b50028fad3616c87c7aa801653232a1361e.1411569661.git.dledford@redhat.com>
Patchwork-id: 96590
O-Subject: [Patch RHEL7 22/29] IB/usnic: Remove unused variable in usnic_debugfs_exit()
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>

The variable qp_grp is initialized but never used otherwise, so remove
the unused variable.

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit d1db47c5eed89a1c8b60e780aeadd870bb0d3894)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_debugfs.c b/drivers/infiniband/hw/usnic/usnic_debugfs.c
index 6cb2e7c..5d13860 100644
--- a/drivers/infiniband/hw/usnic/usnic_debugfs.c
+++ b/drivers/infiniband/hw/usnic/usnic_debugfs.c
@@ -131,13 +131,9 @@ void usnic_debugfs_exit(void)
 
 void usnic_debugfs_flow_add(struct usnic_ib_qp_grp_flow *qp_flow)
 {
- struct usnic_ib_qp_grp *qp_grp;
-
  if (IS_ERR_OR_NULL(flows_dentry))
   return;
 
- qp_grp = qp_flow->qp_grp;
-
  scnprintf(qp_flow->dentry_name, sizeof(qp_flow->dentry_name),
    "%u", qp_flow->flow->flow_id);
  qp_flow->dbgfs_dentry = debugfs_create_file(qp_flow->dentry_name,
-- 
1.7.1